LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Microsoft Graph
+Copyright (c) 2020 Permissions Scrapper
```

Comment:
i'm not sure either of these are right as neither is a company.

Reply:
Resolved